build/cmake/toolkit.cmake

```diff
@@ -141,6 +141,8 @@ if(WXQT)
         set(CMAKE_SHARED_LIBRARY_SUFFIX_C ${CMAKE_SHARED_LIBRARY_SUFFIX})
         set(CMAKE_SHARED_LIBRARY_SUFFIX_CXX ${CMAKE_SHARED_LIBRARY_SUFFIX})
     endif()
+
+    list(APPEND wxTOOLKIT_DEFINITIONS __WXQT__)
```

Comment:
This is/should already be set in
https://github.com/wxWidgets/wxWidgets/blob/0e4a5d9a0d748a4e5ae475e3c83fc988076a3098/build/cmake/toolkit.cmake#L57

Reply:
You're right.